src/components/Cards/ValueCard.tsx

```diff
@@ -19,47 +16,28 @@ export interface IProps {
 }
 export const ValueCard: React.FC<IProps> = ({ name, value, sign, hint, onClick }) => {
   const classes = useStyles()
+  console.log(Number(value))
```

Comment:
Looks fine. Remove this console.log().

You should also configure your identity in git: `git config user.email "MY_NAME@example.com"`

Reply:
ok, i deleted console.log and identified the email. Merge with dev?

Reply:
sure